gulpfile.js

```diff
@@ -25,7 +25,7 @@ gulp.task('lint', ['eslint']);
 gulp.task('default', ['lint']);
 
 gulp.task('gen-prefs', function(cb){
-  var contents = '// Set prefs to use a local content server\n';
+  var contents = '// Set prefs to use a local content server\n'; // eslint-disable-line
```

Comment:
Oh, of course! This doesn't run in the browser, it runs at the command line. Duh.